docs/reference/sso.md

```diff
@@ -0,0 +1,522 @@
+
+Important: new app registrations are hidden to users by default. When you are ready for users to see the app on their My Apps page you can enable it. To enable the app, in the Azure portal navigate to Azure Active Directory > Enterprise applications and select the app. Then on the Properties page toggle Visible to users? to Yes.
+
+#### Add a secret
+Sometimes called an application password, a client secret is a string value your app can use in place of a certificate to identity itself.
+
+Client secrets are considered less secure than certificate credentials. Application developers sometimes use client secrets during local app development because of their ease of use. However, you should use certificate credentials for any of your applications that are running in production.
+
+Steps:
+1. In the Azure portal, in App registrations, select your application.
+2. Select Certificates & secrets > Client secrets > New client secret.
+3. Add a description for your client secret.
+4. Select an expiration for the secret or specify a custom lifetime.
+    1. Client secret lifetime is limited to two years (24 months) or less. You can't specify a custom lifetime longer than 24 months.
+    2. Microsoft recommends that you set an expiration value of less than 12 months.
+5. Select Add.
+6. Record the secret's value for use in your client application code. This secret value is never displayed again after you leave this page.
+
+#### Add a redirect URI
+A redirect URI is the location where the Microsoft identity platform redirects a user's client and sends security tokens after authentication.
+
+You add and modify redirect URIs for your registered applications by configuring their platform settings.
+
+Enter https://lakefs-cloud.us.auth0.com/login/callback or https://lakefs-cloud.eu.auth0.com/login/callback (depends on your organization data location) as your redirect URI.
+
+Settings for each application type, including redirect URIs, are configured in Platform configurations in the Azure portal. Some platforms, like Web and Single-page applications, require you to manually specify a redirect URI. For other platforms, like mobile and desktop, you can select from redirect URIs generated for you when you configure their other settings.
+
+Steps:
+1. In the Azure portal, in App registrations, select your application.
+2. Under Manage, select Authentication.
+3. Under Platform configurations, select Add a platform.
+4. Under Configure platforms, select the web option.
+5. Select Configure to complete the platform configuration.
+
+Once you finish registering lakeFS Cloud with Azure AD, save the **Application (Client) ID**, **Application Secret Value** and send this to Treeverse's team to finish the integration.
+
+## Enterprise
+
+lakeFS Enterprise provides a secondary service, running side-by-side with lakeFS which handles the authentication, this service is called Fluffy.
```

Comment:
Is exposing this internal name deliberate (like [graveler](https://docs.lakefs.io/understand/glossary.html#graveler)) and if so are we going to write it up somewhere?

Reply:
due to the fact this is the only place Fluffy is being used right now, I thought mentioning it here would be sufficient.
do you think we should have a centralized place with our codenames and what service they represent?

Reply:
It's up to you folk if you want to expose it, or just abstract it out in the docs to something like "lakeFS authorisation proxy" (or whatever it is). There are pros and cons to keeping it

Pros
- keeps the software personable and not boring
- less mental translation for devs when writing docs and talking to users

Cons
- depending on its use / surface area, moves the mental workload to users who have to figure out wtf `fluffy` and `graveler` are (but in practice not, because they're relatively low-level things that aren't day-to-day user accessed)

I'd say keep it, but at a minimum add an entry for it to the glossary.